bodyshopbidsdotcom/git-scripts
# Enable metrics on autoscaling group

Depends on https://github.com/bodyshopbidsdotcom/infra/pull/42

## What

By default group metrics are disabled on auto scaling group and terraform has no way to enable those from beanstalk environment. So using aws cli to enable those.

## Testing
1. Go to autoscaling group of any env, click on `Monitoring` tab and you should see option `Enable Group Metrics Collection`
2. Run `git_push_eb stage`
3. Go to autoscaling group of any env, click on `Monitoring` tab and you should see group metrics enabled and `Disable Group Metrics Collection` option
